docs/Embedded/EnkluVolume.md

```diff
@@ -1,16 +1,16 @@
 ---
 id: EnkluVolume
-title: Embedding an Experience
+title: Embedding an Experience in Your Unity Scene
 ---
 
 To embed an Enklu Cloud experience in your Unity project, add an EnkluVolume prefab to your scene. Find it by searching for "EnkluVolume" in the project tab and ensuring the "In Packages" option is selected. Once the prefab is in your scene, select it and log in to your Enklu Cloud account in the inspector tab.
 
 ![Enklu Volume](/img/product/embedded/VolumeImport.gif)
 
-After you've logged in, use the inspector tab to set an existing experience to embed or create a new one. You can embed an entire Enklu experience in your scene or just part of one by selecting or creating an element.
+After you've logged in, use the inspector tab to set up your Enklu Volume. You can embed an entire experience or just part of one. Select an element from the dropdown menus to choose which part.
```

Comment:
I think it's just a singular dropdown menu, right? So maybe this sentence should be "Select an element from the dropdown menu to choose which part of your experience you would like to embed" or something to that effect

Reply:
It _is_ multiple dropdowns, but using "menus" plural does seem like a typo.